geopandas/plotting.py

```diff
@@ -852,6 +852,43 @@ def plot_dataframe(
     return ax
 
 
+if geopandas._compat.PANDAS_GE_025:
+    from pandas.plotting import PlotAccessor
+
+    class GeoplotAccessor(PlotAccessor):
+        """Extend Pandas PlotAccessor to plot GeoDataFrames.
+        The different kinds of plots :
+         - 'geo': Map (default)
+         Pandas Kinds
+         - 'line' : line plot
+         - 'bar' : vertical bar plot
+         - 'barh' : horizontal bar plot
+         - 'hist' : histogram
+         - 'box' : boxplot
+         - 'area' : area plot
+         - 'pie' : pie plot
+         - 'scatter' : scatter plot
+         - 'hexbin' : hexbin plot.
```

Comment:
`kde` is missing. See https://github.com/pandas-dev/pandas/blob/master/pandas/plotting/__init__.py#L1